_docs/sysadmin/configuration/saml_authentication.md

```diff
@@ -272,22 +272,6 @@ To configure your system:
 
 5.  After making the switch to SAML, you will likely want to force
     everyone to re-authenticate with your new Identity Provider
-    (IdP).  This can be done in a few ways:
-
-    * By clearing the session table in the database.
-
-      ```
-      TRUNCATE TABLE sessions;
-      ```
-
-    * Or, by running CONFIGURE_SUBMITTY.py clears the jwt.
-
-      TODO: VERIFY THIS???
-
-
-    * Or, by deleting the jwt secrets file.  It will be recreated
-      (TODO: WHEN?  When INSTALL_SUBMITTY.sh is run???)
-
-      ```
-      rm /usr/local/submity/config/secrets_submitty_php.json
-      ```
\ No newline at end of file
+    (IdP). This will happen automatically after running the
+    CONFIGURE_SUBMITTY.py script. If you make a change and want
+    all users to login again, then just re-run CONFIGURE_SUBMITTY.py.
```

Comment:
Let's not delete the other ways just yet.
re-running CONFIGURE_SUBMITTY.py currently has a few negative side effects (loss of various configuration settings -- I think some are in issues)